massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -922,13 +940,18 @@ fn events_from_switching_blockclique() {
 /// Return a result that should be unwrapped in the root `#[test]` routine.
 fn create_execute_sc_operation(
```

Comment:
It seems the function create_execute_sc_operation_with_datastore was already avail. so maybe we can have only one function here?